content/blog/2022-05-03-e2e-cv-pt1.md

```diff
@@ -0,0 +1,307 @@
+---
+title:
+  'End-to-End Computer Vision API, Part 1: Data Versioning and ML pipelines'
+date: 2022-05-03
+description: >
+
+  In the last few years, training a well-performing Computer Vision (CV) model
+  in Jupyter Notebooks became fairly straightforward if you use pre-trained Deep
+  Learning models and high-level libraries that abstract away much of the
+  complexity (`fastai`, `keras`, `pytorch-lightning` are just a few
+  examples).  The hard part is still incorporating this model into an
+  application that runs in a production environment bringing value to the
+  customers and our business.
```

Comment:
I'm not sure I get this sentence. Do you mean something like this?

```suggestion
  examples).  However it's still hard to integrate these models into a
  production application in a way that brings value to the
  customers and business.
```

And probably adding a sentence about what solution to this problem will be presented would be good.

Reply:
done